src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -472,23 +479,29 @@ bool remove_virtual_functionst::remove_virtual_functions(
 {
   bool did_something=false;
 
-  Forall_goto_program_instructions(target, goto_program)
+  for(goto_programt::instructionst::iterator
+      target = goto_program.instructions.begin();
+      target != goto_program.instructions.end();
+      ) // no ++target
```

Comment:
Nit: this comment is confusing as it looks like `++target`!! Consider `// Don't increment target (why ??)`

Reply:
Done.